public/css/globals/_variables.styl

```diff
@@ -5,7 +5,7 @@
 /* Typefaces */
 
 $font-facit = 'jaf-facitweb', 'Helvetica Neue', Helvetica, Arial, sans-serif
-$font-inconsolata = 'inconsolata', menlo, consolas, monospaced
+$font-inconsolata = 'inconsolata', menlo, consolas, monospaced, monospace
```

Comment:
Should `monospaced` be removed? Doesn't seem to be a valid generic family name: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family